src/dispatch/static/dispatch/src/task/Table.vue

```diff
@@ -136,17 +142,17 @@ export default {
   data() {
     return {
       headers: [
-        { text: "Incident Name", value: "incident.name", sortable: false },
-        { text: "Incident Priority", value: "incident.incident_priority.name", sortable: false },
-        { text: "Incident Type", value: "incident.incident_type.name", sortable: false },
+        { text: "Incident Name", value: "incident.name", sortable: true },
+        { text: "Incident Priority", value: "incident_priority.name", sortable: true },
+        { text: "Incident Type", value: "incident_type.name", sortable: true },
         { text: "Status", value: "status", sortable: true },
-        { text: "Creator", value: "creator", sortable: true },
-        { text: "Owner", value: "owner", sortable: true },
+        { text: "Creator", value: "creator.individual_contact.name", sortable: true },
+        { text: "Owner", value: "owner.individual_contact.name", sortable: true },
```

Comment:
`individual` not `individual_contact`?

Reply:
No, the actual model is individual contact (which is then resolved and used for filtering). We have a disconnect in that the actual model is called `individual_contact` but we refer to it as `individual` in most places.

Reply:
Yeah, it's a little bit confusing.